src/library/mesh_to_grid_map_converter.cpp

```diff
@@ -60,10 +82,14 @@ void MeshToGridMapConverter::meshCallback(
   // Converting from message to an object
   pcl::PolygonMesh polygon_mesh;
   pcl_conversions::toPCL(mesh_msg, polygon_mesh);
+  meshToGridMap(polygon_mesh, mesh_msg.header.frame_id, mesh_msg.header.stamp.toNSec());
+}
 
+bool MeshToGridMapConverter::meshToGridMap(const pcl::PolygonMesh &polygon_mesh, const std::string &mesh_frame_id,
```

Comment:
What is RSL's autoformatter policy? This would break our (google style) max character per line limit of 80 characters. Are you guys using clang-format?

Reply:
Ok, I'll change it to 80 and in the mean time I'll ask Micheal to buy you a big enough screen to fit at least 120 characters.